app/assets/javascripts/Components/annotation_stat_panel.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import ReactTable from 'react-table';
+import { render } from 'react-dom';
+import ReactDOM from 'react-dom';
+
+class AnnotationStatPanel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {}
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  remove_component = (panel) => {
+    ReactDOM.unmountComponentAtNode(panel);
+  }
+
+  columns = [
+    {
+      Header: I18n.t('annotations.used_by'),
+      accessor: row => row['users.first_name'] + ' ' + row['users.last_name'],
+      id: 'user',
+      minWidth: 200,
+      maxWidth: 200,
+      resizeable: false,
+      PivotValue: ({ value }) =>
+        <span>
+          {value}
+        </span>,
+      Cell: row => {
+        return (
+          <div>
+            {row.original['users.first_name'] + ' ' + row.original['users.last_name']}
+          </div>
+        );
+      }
+    },
+    {
+      Header: I18n.t('activerecord.models.submission.one'),
+      accessor: 'groups.group_name',
+      aggregate: (vals, pivots) => {
```

Comment:
For a single-expression lambda, you can do `(vals, pivots) => pivots.length`.

But also, this is a place where some extra text would be nice. E.g., `'Used {pivots.length} times'`